dispatch/apps/api/serializers.py

```diff
@@ -21,11 +21,15 @@ class PersonSerializer(DispatchModelSerializer):
     """
     Serializes the Person model.
     """
+
     class Meta:
         model = Person
         fields = (
             'id',
             'full_name',
+            'slug',
+            'description',
+            'image'
```

Comment:
We should add the `ValidFilename` validator to the image field. Here's an example of how to do that: https://github.com/ubyssey/dispatch/blob/develop/dispatch/apps/api/serializers.py#L55